modules/home/classes/transformations-manager.php

```diff
@@ -15,6 +15,7 @@ class Transformations_Manager {
 		'Create_New_Page_Url',
 		'Filter_Plugins',
 		'Remove_Sidebar_Upgrade_For_Pro_Users',
+		'Generate_Sidebar_Upgrade',
```

Comment:
Can this transformer have a clearer name?

Reply:
@TzviRabinovitch & @hein-obox I already changed name once, please, suggest what you think is suitable